internal/gitinterface/commit.go

```diff
@@ -79,7 +86,60 @@ func VerifyCommitSignature(commit *object.Commit, key *tuf.Key) error {
 
 		return nil
 	case signerverifier.FulcioKeyType:
-		return ErrUnknownSigningMethod // TODO: implement
+		root, err := fulcioroots.Get()
+		if err != nil {
+			return errors.Join(ErrVerifyingSigstoreSignature, err)
```

Comment:
If you don't want to bump to Go 1.20 you could do something like this instead https://go.dev/play/p/L5P9FiIZS07

Reply:
Yeah, I think I'll keep 1.19 around for a bit longer.